.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,75 @@
+# docs https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+
+name: Create a bug report
+description: Errors and regression reports with complete reproducing test cases and/or stack traces.
+labels: [requires triage]
+body:
+  - type: markdown
+    attributes:
+      value: "
+**GUIDELINES FOR REPORTING BUGS**
+
+If you are new to SQLAlchemy bug reports, please review our many examples
+of [well written bug reports](https://github.com/sqlalchemy/sqlalchemy/issues?q=is%3Aissue+label%3A%22great+mcve%22).   Each of these reports include the following features:
+
+1. a **succinct description of the problem** - typically a line or two at most
+
+2. **succinct, dependency-free code which reproduces the problem**, otherwise known as a [Minimal, Complete, and Verifiable](https://stackoverflow.com/help/mcve) example.
+
+3. **complete stack traces for all errors - please avoid screenshots, use formatted text inside issues**
+
+4. Other things as applicable:   **SQL log output**, **database backend and DBAPI driver**,
+   **operating system**, **comparative performance timings** for performance issues.
+"
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: To Reproduce
+      description: "
+Provide your [Minimal, Complete, and Verifiable](https://stackoverflow.com/help/mcve) example here."
```

Comment:
I would add to the description:

If you need help creating one, you can model yours after the MCV code shared in one of our previous [well written bug reports](https://github.com/sqlalchemy/sqlalchemy/issues?q=is%3Aissue+label%3A%22great+mcve%22)

Reply:
Its referenced at the top, but we can add that here also.

Reply:
Yeah, I basically just copy/pasted your line to repeat it there. People are bad about code samples, so repeating it and just being "YOU CAN USE ANOTHER FORKING PERSON'S EXAMPLE CODE AS A TEMPLATE" is helpful.

Reply:
it was my reasoning too. will update